OpenChange/MAPIStoreMessage.m

```diff
@@ -178,56 +178,109 @@ - (NSDictionary *) _convertRecipient: (struct mapistore_message_recipient *) rec
                           andColumns: (struct SPropTagArray *) columns
 {
   NSMutableDictionary *recipientProperties;
+  enum MAPITAGS prop_tag;
+  char *displayName = NULL;
+  char *email = NULL;
+  enum MAPI_OBJTYPE object_type = 0;
+  char *smtpAddress = NULL;
   SOGoUser *recipientUser;
   NSUInteger count;
-  id value;
+  NSString *recipientEmail = nil;
+  NSString *recipientFullName = nil;
+  NSString *value;
```

Comment:
This is incorrect when the property value is different from Unicode and according to Line 216 and Line 205 this is happening. Please, keep the declaration to `id value` svp.